changes/4112.added

```diff
@@ -0,0 +1,7 @@
+- Added `invoke unittest-parallel`.
+    - Test outputs are stored in `.test-results` directory.
+    - There is a `tests-times.json` file used for distributing tests. Use `invoke sum-tests-times` to update it.
+- Added `invoke ps` to display docker compose services.
+- Added `invoke export-docker-compose` to export docker compose configuration to `compose.yaml` file.
+- Added `invoke logs` to display docker compose logs.
+- Fixed `api-status` API call when using higher Redis DB index.
```

Comment:
towncrier automatically splits each line in this file into a separate bullet-list item.

```suggestion
Added `invoke unittest-parallel`, which runs unit tests in parallel with results written to `.test-results` directory.
Added `test-times.json` file which is used to distribute tests when running in parallel. This file can be updated by running `invoke sum-tests-times`.
Added `invoke ps` to display docker compose services.
Added `invoke export-docker-compose` to export docker compose configuration to `compose.yaml` file.
Added `invoke logs` to display docker compose logs.
Fixed `api-status` API call when using higher Redis DB index.
```

Also if that last item is a bugfix, it should be split into a separate `4112.fixed` file for consumption by towncrier.

Reply:
Fixed